requirements.txt

```diff
@@ -1,5 +1,6 @@
 dependency-injector>=3.14
 uvloop>=0.14
 aiohttp[speedups]>=3.6
+aioredis==1.3.0
```

Comment:
Let's use `>= 1.3` version. It would be nice to continuously update dependencies for now.